gensim/models/base_any2vec.py

```diff
@@ -208,12 +231,23 @@ def _worker_loop(self, job_queue, progress_queue):
             for callback in self.callbacks:
                 callback.on_batch_begin(self)
 
-            tally, raw_tally = self._do_train_job(data_iterable, job_parameters, thread_private_mem)
+            stats_tuple = self._do_train_job(
+                data_iterable, job_parameters, thread_private_mem)
```

Comment:
I think there is no need to break the line here. We have a hard limit of 120 characters per line in gensim, and this is well within that limit.